jupyter_server/event_schemas/kernel_actions/v1.yaml

```
@@ -0,0 +1,71 @@
+"$id": https://events.jupyter.org/jupyter_server/kernel_actions/v1
+version: 1
+title: Kernel Manager activities
+personal-data: true
+description: |
+  Record events of a kernel manager.
+type: object
+required:
+  - action
+  - kernel_id
+  - msg
```

I feel we should be consistent (the `gateway_client` schema uses `message`) and this should be expanded:
```suggestion
  - message
```

---

This is actually an (annoying and undocumented) limitation of jupyter_events. 

"message" is a special key that cannot be used in an event schema, because Python's underlying `logging` library mangles this key. See this chunk: https://github.com/jupyter/jupyter_events/blob/00d20622d413ea8b7ed950377f513d3a3db4d84c/jupyter_events/logger.py#L152-L159

We should add some logic to the jupyter_event's metaschema to exclude this key.

---

Interesting.  That's a drag since I'm sure `message` will be a common property of sorts.  Does the schema registration raise an exception when a "candidate schema" contains a `message` property?

---

No it doesn't. We need to add this to the metaschema. 

We could probably add some logic to get around this issue in jupyter_events, I just haven't had the change to try this yet.

---

If the events PR becomes too involved we could use a different name.  Some suggestions: `text`, `event_message`, `body`, `content`, ...?  I kinda like `content`.

---

I think, `msg` should be okay compare to other key words. `body` make sense to me but it confuses us with REST `body` which is little different. Following are a few more options.
`comment`
`notification`